iris_grib/__init__.py

```diff
@@ -1024,7 +1024,7 @@ def save_grib2(cube, target, append=False, **kwargs):
     See also :func:`iris.io.save`.
 
     """
-    messages = as_messages(cube)
+    messages = (message for _, message in save_pairs_from_cube(cube))
```

Comment:
@dkillick As you can see, there is [no test coverage](https://coveralls.io/builds/6731708/source?filename=iris_grib%2F__init__.py#L996) for `save_grib2` ... tsk, tsk, tsk!

Guess I'd better create some :wink: